### PR TITLE
docs(basics/controller.md): [translate] Done.

### PR DESCRIPTION
**Pull request by lslxdx**
'basics/controller.md' is translated and waiting for being proofread.

##### Checklist
- [x] documentation is changed or added

### CODE REVIEW

**Comment by lslxdx** [comment]
The translation is done. Could someone help to proofread it?

P.S:
This PR is created from an [old work process](https://github.com/eggjs/egg/issues/363#issuecomment-300387067) and new translations WILL NOT be done in this way.

**Comment by popomore** [comment]
@lslxdx who will proofread in SwiftGG team?

**Comment by lslxdx** [comment]
@popomore It's a volunteer that proofreads it not a certain person.

**Comment by popomore** [comment]
OK, I'll wait one to proofread it

**Comment by popomore** [comment]
https://github.com/eggjs/egg/issues/363

File: docs/source/en/basics/controller.md [hunk 1]
```diff
@@ -0,0 +1,846 @@
...
+`context.set(key, value)` sets one response header and `context.set(headers)` sets many in one time.
+
+```js
+// app/controller/api.js
+exports.show = function* (ctx) {
+  const start = Date.now();
+  ctx.body = yield ctx.service.post.get();
+  const used = Date.now() - start;
+  // set one response header
+  ctx.set('show-response-time', userd.toString());
```
Comment (coogleyao) [inline docs/source/en/basics/controller.md]: use `used`, not `userd`.

Comment (lslxdx) [inline docs/source/en/basics/controller.md]: correct.

It's copied from the Chinese version.

Comment (coogleyao) [inline docs/source/en/basics/controller.md]: 😄 Don't worry, I have fixed that.

**Pull request merged**